nss-day-cohort-29/reactive-nutshell-copperden
# Register

# Description
User can now register and it makes a post to the database and grabs the current id for session storage later on.

Also login now redirects the user to "/news" in the url.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout register`
3. Register a new user and check the console to see if the users name, email, and ID are displayed in your console.

# Checklist:
